### ISSUE TITLE
Config breaks when selecting 'store as project file'.

### ISSUE THREAD

**Issue by NathanielArmer** [GoogleCloudPlatform/cloud-code-intellij#2969]
### Expected Behavior

Create a Cloud Run Config (e.g. to run locally, or deploy to cloud run).  Click store as project file. 

I expect a project file to be created that contains the config, and can be used to run/deploy my cloud run project.

### Actual Behavior

A project file is created, but the 'builder' section of the config is lost, breaking the config.

### Additional Information

- IDE type (e.g. IntelliJ, Pycharm): Rider.
- IDE version: 2021.1.2
- Cloud Code version (Settings > Plugins > Cloud Code):  21.6.1
- Cloud SDK (Settings > Cloud Code > Cloud SDK)
  - Are you allowing the plugin to manage the Cloud SDK: Not Sure.
  - Version of the Cloud SDK: Don't seem to have one?
  - ```> gcloud --version
          Google Cloud SDK 345.0.0
          bq 2.0.69
          core 2021.06.11
          gsutil 4.63
- Operating System: OSX 11.4 (20F71)

**Comment by grmoon** [comment]
Thanks for the report @NathanielArmer! We're actively looking into this and will update this issue with further questions/updates as necessary

**Comment by grmoon** [comment]
Kamino cloned this issue to [GoogleCloudPlatform/cloud-code-intellij-internal](https://github.com/GoogleCloudPlatform/cloud-code-intellij-internal/issues/4390)

**Comment by grmoon** [comment]
@NathanielArmer we have identified the root cause of this issue as an issue with the Rider platform code. We have filed an issue with JetBrains that can be tracked [here](https://youtrack.jetbrains.com/issue/RIDER-66002). 

Pending their response/fix a workaround that we have found is to save the run configuration as a project file to a location other than the default `.run` directory. Specifically, you can save the run configuration to a **non-hidden** directory in your project and things should work as expected.

**Comment by NathanielArmer** [comment]
@grmoon Thanks so much!

I tried making my configs in a non-hidden directory and it appears to work! Unfortunately Rider can't find them after closing the solution... but that's likely an entirely separate issue.

I appreciate your efforts, and look forward to the `.run` dir fix when the JetBrains teams has a resolution.

**Comment by ivanporty** [comment]
@grmoon https://youtrack.jetbrains.com/issue/RIDER-66002 marked `Fixed`, anything changed for us in this issue?

**Comment by grmoon** [comment]
> @grmoon https://youtrack.jetbrains.com/issue/RIDER-66002 marked `Fixed`, anything changed for us in this issue?

based on the resolution in the YouTrack issue, it looks like we'll need to implement the new `com.intellij.baseDirectorySuggester` extension point for 2021.3 and test in Rider before we can mark it as resolved.

**Comment by emmanuelbaah** [comment]
Setting priority to `p3`. Will triage as part of backlog grooming

**Comment by peterlin741** [comment]
The team is still looking into this

**Comment by ruomengz** [comment]
Based on further investigation, this issue is not a high priority. WIll close for now.
